tox.ini

```diff
@@ -0,0 +1,16 @@
+[tox]
+envlist = py{27,36,37}
+skipsdist = True
+
+[testenv]
+usedevelop = True
+deps =
+   pytest
+   coverage
+setenv =
+    PYTHONPATH={toxinidir}
+commands =
+    coverage run --source rdbsync -m py.test
+    coverage xml
+
```

Comment:
We need: 
`passenv = HOME` 

somewhere in here so we can find our directory when running tests in openshift